purestorage/purestorage.py

```diff
@@ -2001,7 +2001,7 @@ def test_snmp_manager(self, manager):
     # Note: These methods only work with REST API 1.2 and later
     #
 
-    def connect_array(self, address, connection_key, connection_type, **kwargs):
+    def connect_array(self, management_address, connection_key, connection_type, **kwargs):
         """Connect this array with another one.
 
         :param address: IP address or DNS name of other array.
```

Comment:
Can you update the docs here for the new name too? Should just mean adjusting the param and type names here from address to management_address